lib/git/pr/release/util.rb

```diff
@@ -157,7 +157,7 @@ def merge_pr_body(old_body, new_body)
           merged_body = pr_body_lines.join("\n")
           check_status.each { |issue_number, check_value|
             say "Update pull-request checkbox \##{issue_number} to #{check_value}.", :trace
-            merged_body.gsub! /^- \[ \] \##{issue_number}/, "- [#{check_value}] \##{issue_number}"
+            merged_body.gsub! /^- \[ \] \##{issue_number}\s/, "- [#{check_value}] \##{issue_number}\s"
```

Comment:
```suggestion
            merged_body.gsub! /^- \[ \] \##{issue_number}\b/, "- [#{check_value}] \##{issue_number}"
```

I think `\b` for word boundaries might be more appropriate to avoid unintended matches.

Reply:
@onk 
Thank you. Indeed, the content you suggested seems better!